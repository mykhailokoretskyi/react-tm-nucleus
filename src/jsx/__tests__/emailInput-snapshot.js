import React from 'react';
import EmailInput from '../emailInput';
import renderer from 'react-test-renderer';

describe('TextInput', () => {
    it('mocks match snapshots', () => {
        const component = renderer.create(
            <EmailInput value="" placeholder="enter rhee..." />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('with value matches snapshot', () => {
        const component = renderer.create(
            <EmailInput value="2sdf" required={true} />
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
