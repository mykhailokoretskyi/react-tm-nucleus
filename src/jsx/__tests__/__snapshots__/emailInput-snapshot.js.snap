exports[`TextInput mocks match snapshots 1`] = `
<div
  className="input-group">
  <input
    checked={false}
    className="input-group__text-field "
    disabled={false}
    id=""
    onBlur={[Function bound _onBlur]}
    onChange={[Function bound _onChange]}
    onClick={[Function bound _onClick]}
    onFocus={[Function bound _onFocus]}
    onMouseEnter={[Function bound _onMouseEnter]}
    onMouseLeave={[Function bound _onMouseLeave]}
    placeholder="enter rhee..."
    required={false}
    type="email"
    value="" />
  <label
    className="input-group__label"
    dangerouslySetInnerHTML={
      Object {
        "__html": " "
      }
    }
    htmlFor="" />
</div>
`;

exports[`TextInput with value matches snapshot 1`] = `
<div
  className="input-group">
  <input
    checked={false}
    className="input-group__text-field has-content"
    disabled={false}
    id=""
    onBlur={[Function bound _onBlur]}
    onChange={[Function bound _onChange]}
    onClick={[Function bound _onClick]}
    onFocus={[Function bound _onFocus]}
    onMouseEnter={[Function bound _onMouseEnter]}
    onMouseLeave={[Function bound _onMouseLeave]}
    required={true}
    type="email"
    value="2sdf" />
  <label
    className="input-group__label"
    dangerouslySetInnerHTML={
      Object {
        "__html": " <span class=\"input-group__required\">*</span>"
      }
    }
    htmlFor="" />
</div>
`;
